goas/cl/cl/src/input.rs

```diff
@@ -20,30 +21,44 @@ pub struct Input {
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
 pub struct InputWitness {
     pub note: NoteWitness,
+    pub utxo_balance_blinding: BalanceWitness,
+    pub balance_blinding: BalanceWitness,
```

Comment:
I think this deserves a comment with some explanation

Reply:
Fixed in the simple transfer branch: https://github.com/logos-co/nomos-pocs/pull/7/commits/c5bcf3ebe709f16dea2d723be747f6a359817fe5#diff-5497e9582a968c88fc1e9ffbe619c6ddfcd435578d107b7c15cb82116e347ba0L81